addon/webextension/selector/callBackground.js

```diff
@@ -0,0 +1,17 @@
+/* globals browser */
```

Comment:
I believe @Standard8 was still working through some install issues with the Firefox standard eslint package, but that got shelved for now since there are higher priority things.

I'd like to adopt the Firefox rules, and fix up these declarations along the way, but it seems easiest and least intrusive to do it in one big batch.